tidb-cloud/serverless-private-link-connection-to-aws-confluent.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Connect to AWS Confluent via a Private Link Connection
+summary: Learn how to connect to an AWS Confluent instance using an AWS Confluent Endpoint Service private link connection.
+---
+
+# Connect to Confluent Cloud via a Private Link Connection
+
+This document describes how to connect to a Confluent Cloud Dedicated cluster on AWS using an AWS Endpoint Service private link connection.
+
+> **Note**
+>
+> - Only Confluent Cloud Dedicated clusters on AWS are supported.
+
+## Prerequisites
+   
+- You have a Confluent Cloud account.
+
+- Confirm that your {{{ .essential }}} is active in AWS. Retrieve and save the following details for later use:
+
+    - Account ID
+    - Availability Zones (AZ)
+
+To view the the AWS account ID and available zones, do the following:
+
+1. In the [TiDB Cloud console](https://tidbcloud.com), navigate to the cluster overview page of the TiDB cluster, and then click **Settings** > **Networking** in the left navigation pane.
+2. On the **Private Link Connection For Dataflow**, click **Create Private Link Connection**.
+3. You can find the AWS account ID and available zones.
+
+## Step 1. Set up a Confluent Cloud network
+
+Identify a Confluent Cloud network that you want to use, or [create a new Confluent Cloud network on AWS](https://docs.confluent.io/cloud/current/networking/ccloud-network/aws.html#create-ccloud-network-aws).
+
+The Confluent Cloud network must meet the following requirements:
+
+- Type: the network must be a **PrivateLink** network.
+- Region match: the instance must reside in the same AWS region as your {{{ .essential }}} cluster.
+- AZ (Availability Zone) availability: the availability zones must overlap with those of your {{{ .essential }}} cluster.
+
+To get the unique name of the Confluent Cloud network:
+
```

Comment:
Please specify where to perform the following steps.

Reply:
yes，it is on the Confluent Cloud console. have added this